src/observers/stores/duckdb.py

```diff
@@ -58,7 +58,17 @@ def add(self, record: "Record"):
             if record_dict[json_field]:
                 record_dict[json_field] = json.dumps(record_dict[json_field])
 
+        for image_field in record.image_fields:
+            if record_dict[image_field]:
+                record_dict[image_field] = record_dict[image_field].tobytes()
```

Comment:
what if we store it like this:

```
STRUCT(bytes BLOB, path VARCHAR)
```

with path so we have the filename next to the bytes since it's what we do on datasets as well